classes/webservice/WebService.inc.php

```diff
@@ -1,349 +0,0 @@
-<?php
```

Comment:
I'm happy to see these unused files go, but could you submit that as a separate pull request?